internal/web/server.go

```diff
@@ -549,7 +581,7 @@ func (s *server) kubeconfigPage(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *server) enrichWithErrorAndWarnings(ctx context.Context, data map[string]any, err error) map[string]any {
-	cacheBustStr := fmt.Sprintf("%d", rand.Int())
+	cacheBustStr := config.Version // fmt.Sprintf("%d", rand.Int())
```

Comment:
Please remove this commented out code. Additionally, with this change, I think setting the value again below is redundant, since the clientVersion is the same as config.Version.